requirements.txt

```diff
@@ -15,10 +16,11 @@ psycopg2==2.7.1
 pyparsing==2.2.0
 pyScss==1.3.5
 PyYAML==3.12
+redis==2.10.5
 requests==2.13.0
 scipy==0.19.0
 six==1.10.0
 SQLAlchemy==1.1.6
 webassets==0.12.1
 Werkzeug==0.12.1
-xlrd==1.0.0
\ No newline at end of file
+xlrd==1.0.0
```

Comment:
It's cause of a missing newline at EOF (usually, it's an editor setting whether or not you want a newline at the end of a file).

We use [EditorConfig](http://editorconfig.org/) in this project, so if you install EditorConfig for your editor, it'll automatically use the right conventions (due to this file / setting: https://github.com/anishathalye/gavel/blob/master/.editorconfig#L6).